syntax = "proto3";

service ImageClassification {
  rpc ClassifyImage (ImageRequest) returns (ImageResponse);
}

message Prediction {
  string name = 1;
  string clase = 2;
  float score = 3;
}

message ImageRequest {
  string image_path = 1;
}

message ImageResponse {
  repeated Prediction predictions = 1;
}
